src/app/services/export-excel/excel-export-sample-1/excel-export-sample-1.component.ts

```diff
@@ -27,6 +30,12 @@ export class ExcelExportSample1Component implements OnInit {
 
   public ngOnInit(): void {
     this.data = INVOICE_DATA;
+    this.expr = [
```

Comment:
Use more meaningful name.
```suggestion
    this.groupExpressions = [
```